jetty-start/src/main/java/org/eclipse/jetty/start/StartArgs.java

```diff
@@ -1505,6 +1505,12 @@ public void setProperty(String key, String value, String source)
             {
                 JavaVersion ver = JavaVersion.parse(value);
                 properties.setProperty("java.version.platform", Integer.toString(ver.getPlatform()), source);
+
+                // features built into java.
+                properties.setProperty("java.feature.alpn", Boolean.toString(ver.getPlatform() >= 9), source);
+                properties.setProperty("java.feature.jpms", Boolean.toString(ver.getPlatform() >= 9), source);
+                properties.setProperty("java.feature.loom", Boolean.toString(ver.getPlatform() >= 16), source);
```

Comment:
As discussed, I won't add features that we don't use -- keep only `java.feature.alpn`.
For example, loom won't be available in 16 AFAIK, and other features may be available only with `--enable-preview` so it's not only the java version, we may need to sniff the existence of classes/methods, etc.

Reply:
Removed loom entirely.
Left jpms (in 9.4.x) as it has a use there. (no valid use in 10.0.x+)